src/plugins/data_views/common/data_views/abstract_data_views.ts

```diff
@@ -149,6 +149,11 @@ export abstract class AbstractDataView {
             hasAttrs = true;
           }
 
+          if (value.customDescription) {
+            attrs.customDescription = value.customDescription;
+            hasAttrs = true;
+          }
+
```

Comment:
When in Kibana do we initialize data views with `spec.fields`?

Reply:
I'm honestly not sure about this and need to do some research.

Reply:
This code path is used when a DataView or DataViewLazy is created and a spec is passed that includes a field list. The spec includes duplicate information since it includes a field list - field attributes aren't stored on the field list since it may change AND we don't save the field list. However, we have code that uses a spec that only gets references to fields and not the rest of the data view so this duplication is necessary. Its harmless when serializing a data view but can be confusing when importing a data view spec since the duplicate data might not be in agreement. 

We should add a comment above `extractedFieldAttrs` that explains that its importing field attributes when a data view is imported from a spec and those attributes aren't provided in the fieldAttrs.